### PR TITLE
First pass on 08-CI

### PR DESCRIPTION
**Pull request by radifar**
Add new line to each end of sentence, fix typos, change (legacy) Travis to GitHub Actions. Fix YAML highlighting.

I also added TODO at line 441 that needs to be addressed later.

### CODE REVIEW

File: _episodes/08-CI.md [hunk 1]
```diff
@@ -475,73 +413,57 @@ new test.
...
 <center><img src="../fig/08_CI/codecov_screenshot.png"></center>
 <br />
 
-Red areas represent code that is poorly covered by our tests. On the right, you
-see the latest commits and can click on each one of them to see how they changed
-overall coverage of the project. It is possible, although outside the scope of
-this episode, to setup Travis to fail if a commit lowers the coverage of our
-codebase. The bottom of the page shows a simple file-browser like interface,
-much like the HTML page we obtained before.
+[//]: # TODO: Check if the following feature work in GitHub Action, this paragraph is the legacy from Travis CI
```
Comment (janash) [inline _episodes/08-CI.md:441]: Good TODO. I don't think I've done this yet. Please add this to our shared doc to address in the next meeting!

Comment (radifar) [reply _episodes/08-CI.md:441]: Sure!

File: _episodes/08-CI.md [hunk 2]
```diff
@@ -25,56 +25,40 @@ keypoints:
...
-Using any of these services is essentially getting access to empty remote
-computers that we can configure as we wish. In order to test our projects using
-Travis, we will have to set up the correct environment and tell our CI service
-what to do.
+continuous integration (CI) is the practice of merging all developers' working copies of code to a shared mainline, several times a day.
+In other words, CI refers to the coupling of version control and unit testing in an automated way.
+Typically, CI runs either when you commit new code to your project, or beforehand, when you try to merge experimental code into the main repository (through a pull request).
+CI is useful not only for catching bugs before they reach your fellow developers and your end users, but also for expanding your testing to platforms that are not available to you.
+
+Most CI services are tightly integrated with GitHub. In this episode, we will be using [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions),
```
Comment (eirrgang) [inline _episodes/08-CI.md:33]:
```suggestion
Most CI services are tightly integrated with GitHub.
In this episode, we will be using [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions),
```

File: _episodes/08-CI.md [hunk 3]
```diff
@@ -25,56 +25,40 @@ keypoints:
...
-do so by editing these files. Though not included with the cookiecutter, you can
-also have an [issue
-template](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository).
+All of GitHub Actions' settings are stored in the file `.github/workflows/CI.yaml`.
+Note the leading period on the `.github` folder, which makes this a hidden folder in Linux and Mac OS.
+The `.github` file stores information which is relevant for GitHub in general.
+For example, you will also see in this folder a file called `CONTRIBUTING.md` which contains contributing guidelines
+(the information in this file is displayed to someone when they contribute to your project).
+The file `PULL_REQUEST_TEMPLATE.md` is the file which gives the template which users see when they open a pull request on your repository.
+If you wanted to change the template or contributing text, you would do so by editing these files. Though not included with the cookiecutter, you can also have
```
Comment (eirrgang) [inline _episodes/08-CI.md:47]:
```suggestion
If you wanted to change the template or contributing text, you would do so by editing these files.
Though not included with the cookiecutter, you can also have
```

**Review by eirrgang — CHANGES_REQUESTED** [review]
I would want to take a closer look at the GitHub Actions guidelines before merging this.

**Pull request merged**